filebeat/module/kafka/log/_meta/fields.yml

```diff
@@ -17,7 +17,8 @@
       description: >
         Component the log is coming from.
     - name: class
-      type: text
+      # Is there a reaons this would not be an exact search?
+      type: keyword
```

Comment:
No, keyword is perfect.

Reply:
chaning

Reply:
https://github.com/elastic/beats/pull/10398